src/streamsync/command_line.py

```diff
@@ -31,21 +34,29 @@ def main():
     absolute_app_path = _get_absolute_app_path(
         args.path) if args.path else None
     host = args.host if args.host else None
+    api_key = args.api_key if args.api_key else None
 
-    _perform_checks(command, absolute_app_path, host, enable_remote_edit)
-    _route(command, absolute_app_path, port, host, enable_remote_edit)
+    _perform_checks(command, absolute_app_path, host, enable_remote_edit, api_key)
+    _route(command, absolute_app_path, port, host, enable_remote_edit, api_key)
 
-def _perform_checks(command: str, absolute_app_path: str, host: Optional[str], enable_remote_edit: Optional[bool]):
+def _perform_checks(command: str, absolute_app_path: str, host: Optional[str], enable_remote_edit: Optional[bool], api_key: Optional[str] = None):
     is_path_folder = absolute_app_path is not None and os.path.isdir(absolute_app_path)
 
-    if command in ("run", "edit") and is_path_folder is False:
+    if command in ("run", "edit", "deploy") and is_path_folder is False:
         logging.error("A path to a folder containing a Streamsync app is required. For example: streamsync edit my_app")
         sys.exit(1)
 
     if command in ("create") and absolute_app_path is None:
         logging.error("A target folder is required to create a Streamsync app. For example: streamsync create my_app")
         sys.exit(1)
 
+    if command in ("deploy") and api_key is None:
+        logging.info("An API key is required to deploy a Streamsync app.")
+        api_key = input("Enter your API key: ")
```

Comment:
Nice, thanks for making that change. I think that'll display the token, right?

I found this which should work, can you test?

https://docs.python.org/3.9/library/getpass.html